src/services/FirebaseService.ts

```diff
@@ -1,4 +1,3 @@
-import { Address } from "services/EthereumService";
```

Comment:
- Reason1: I cannot import FirebaseService easily into Cypress, because it pulls in EthService, so quickest solution was to remove the dep on EthService.
- Reason2: Easiest way, to not have too much trouble with webpack
        - In general, reducing the import footprint of each file might be a good idea overall